go.mod

```diff
@@ -20,6 +20,8 @@ require (
 	github.com/golang/protobuf v1.4.3
 	github.com/graymeta/stow v0.2.7
 	github.com/hashicorp/golang-lru v0.5.4
+	github.com/lyft/flyteplugins v0.5.26
```

Comment:
this is circular dependency and seems odd. We should not add flyteplugins to flytestdlib. why was this needed?

side note - @katrogan is working on removing lyft/api from the dependency chain.

Reply:
Hi @kumare3 ahah yes, I got puzzled as well but haven't yet investigated. Basically, this was just the product of running `go mod tidy`. So there's some dep here that brings it in.